pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -94,12 +94,12 @@ func toSelectableFields(binding *servicecatalog.Binding) fields.Set {
 }
 
 // GetAttrs returns labels and fields of a given object for filtering purposes.
-func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
+func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, bool, error) {
```

Comment:
@pmorie as @MHBauer pointed, the change just follows the breaking change in `AttrFunc` method https://github.com/kubernetes/apiserver/blob/master/pkg/storage/selection_predicate.go#L25

And the method doesn't have any description in many implementations: [podtemplate/strategy.go#L92](https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/podtemplate/strategy.go#L92), [replicationcontroller/strategy.go#L148](https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/replicationcontroller/strategy.go#L148), [daemonset/strategy.go#L134](https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/extensions/daemonset/strategy.go#L134).

I agree that it is not well documented, but for Service Catalog just being compliant with Kubernetes core code should be enough. It makes it easier to sync up all the breaking changes as needed.
 
> Looking through git history the bool is "the uninitialized flag".

it **_is_** the "uninitialized flag", as mentioned in the comment to the `AttrFunc` you quoted above.
I would keep it as it is.